mantidproject/mantid
# Updated Chronus TF detector groupings

**Description of work.**
This updates the default grouping of CHRONUS in TF asymmetry mode to use a new set of defaults provided by the scientists.



**To test:**
 * Try loading CHRONUS1000 (a longitidunal run) and check that two groups and a pair are made.
 * Try loading CHRONUS5000 (a transverse tun) check that 8 groups are produced by default.



*There is no associated issue.*



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
